src/app/components/window-terminal/window-terminal.component.ts

```diff
@@ -46,6 +46,10 @@ export class WindowTerminalComponent implements OnInit, OnDestroy {
       label: "Paste",
       click: () => this.paste()
     }));
+    this.ctxMenu.append(new MenuItem({
+      label: "Save Buffer",
```

Comment:
please use single quotes. also, I would named label `Save Screen Buffer` so users get an idea what actually this is.